pyfiglet/__init__.py

```diff
@@ -759,6 +767,28 @@ def getFonts(self):
         return self.Font.getFonts()
 
 
+def __color_to_ansi(color, isBackground):
+    if not color:
+        return ''
+    if color.count(';') > 0 and color.count(';') != 2:
+        raise InvalidColor('Specified color \'{}\' not a valid color follow R;G;B format')
+    elif color.count(';') == 0 and color not in COLOR_CODES:
+        raise InvalidColor('Specified color \'{}\' not found in ANSI COLOR_CODES list'.format(color))
+    # validate entry
+    try:
+        ascii_code = COLOR_CODES[color]
+        if isBackground:
+            ascii_code += 10
+    except KeyError:
+        ascii_code = 48 if isBackground else 38
+        ascii_code = '{};2;{}'.format(ascii_code, color)
```

Comment:
This needs to be renamed to `ansi_code`.

Reply:
ups, again :D